content/blogposts/2023/cody-vscode-0-14-release.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Cody for VS Code v0.12 release"
+authors:
+  - name: Kalan Chan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+  - name: Tim Lucas
+    url: https://handbook.sourcegraph.com/team/#tim-lucas
+  - name: Beatrix Woo
+    url: https://handbook.sourcegraph.com/team/#beatrix-woo  
+publishDate: 2023-10-04T10:00-07:00
+description: "inline /doc generation, new custom command modes, smart selection context, and saving chat code to files."
+tags: [blog]
+slug: "cody-vscode-0-14-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/cody-vscode-0.14.0-og-image.jpg
+socialImage: https://storage.googleapis.com/sourcegraph-assets/cody-vscode-0.14.0-og-image.jpg
+---
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v0.14 is now available and includes improvements to the `/doc` and `/test` commands, a new mode configuration for custom commands, and experimental smart selection.
+
+## Inline Documentation with /doc
+
+Previously when using the `/doc` command, the generated documentation would be shown in the chat view. It will now directly insert the documentation text right above the selected code in your editor. This makes it easier to quickly add documentation stubs as you code.
+
+<!-- TODO add example screenshot here -->
+
+## Experimental smart selection
+
+We've started work on a smart selection capability that will automatically determine what code to run the `/doc` and `/test` commands on based on heuristics like scope and recent activity. This will remove the need to manually highlight code before generating documentation or tests.
+
+Smart selection is still experimental but we're excited about the potential to streamline workflows. 
```

Comment:
It's okay to cover behind the scenes, but "We've started work" sounds like maybe there's no changes in this release and it's an FYI. And experimental things are normally opt-in and buggy, and we haven't pointed out how you see/use this new feature, or how you avoid this new feature if you don't want the bugs.

Are there caveats to this feature we should point out? Language support? Known bugs? Do we want feedback if people find it's not working well for them?

What about something more like…

```suggestion
## Smart selection for `/doc` and `/test`

You can now use the `/doc` and `/test` commands without selecting any code first, and Cody will use the position of your cursor to guess out what you're wanting to document or code.

(image)

This feature is still under development, and doesn't yet handle cases xyz. Please give it a test and provide feedback.
```